src/components/EditProject/EditProject.js

```diff
@@ -0,0 +1,440 @@
+import Button from '@material-ui/core/Button';
+// import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
+import PublishIcon from '@material-ui/icons/Publish';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Grid from '@material-ui/core/Grid';
+import Radio from '@material-ui/core/Radio';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import SourceList from '../SourceList/SourceList';
+import UploadSource from '../UploadSource/UploadSource';
+import BibleBooks from '../BibleDropDown/BibleBooks';
+import { API } from '../../store/api';
+import SnackBar from '../SnackBar/SnackBar.js';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Divider from '@material-ui/core/Divider';
+import { makeStyles } from '@material-ui/core/styles';
+import BibleBookTable from './BibleBookTable';
+
+const useStyles = makeStyles({
+  gridLeft: {
+    // paddingLeft: '80px',
+    alignItems: 'center',
+    fontSize: '16px',
+    marginBottom: '5px',
+  },
+  gridRight: {
+    alignItems: 'center',
+    marginBottom: '5px',
+  },
+  dataLearn: {
+    paddingLeft: '80px',
+    alignItems: 'center',
+    marginTop: '20px',
+    fontSize: '14px',
+  },
+  checkbox: {
+    marginTop: '20px',
+  },
+  button: {
+    paddingLeft: '20px',
+    marginTop: '20px',
+    marginBottom: '20px',
+  },
+  // MuiDialogPaperWidthSm: {
+  //   height: '600px',
+  // },
+  active: {
+    marginTop: '40px',
+  },
+});
+
+export default function EditProject(props) {
+  const [open, setOpen] = React.useState(false);
+  const [selectedValue, setSelectedValue] = React.useState('a');
+  const [selectSourceLanguage, setSelectSourceLanguage] = React.useState('');
+  const classes = useStyles(props);
+  const [bookName, setBookName] = useState([]);
+  const [selectedFiles, setSelectedFiles] = React.useState('');
+  const [responseStatus, setResponseStatus] = React.useState([]);
+  const [loading, setLoading] = React.useState('');
+  const [disableButton, setDisableButton] = React.useState(false);
+  const [bookList, setBookList] = React.useState([]);
+  const [projectBooks, setProjectBooks] = useState(['gen', 'exo']);
+  const [sourceBooks, setSourceBooks] = useState();
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  useEffect(() => {
+    // callFn();
+    API.get('bibles/en_KJV_1_bible/books')
```

Comment:
en_KJV_1_bible should not be hard coded as when we change the source the booklist will not get updated. 
breaking change, selectSourceLanguage can be used.